sources/packages/web/src/views/aest/StudentApplicationView.vue

```diff
@@ -15,7 +15,7 @@
       Student Application Details
       {{
         applicationDetail.applicationNumber
-          ? " - " + applicationDetail.applicationNumber
+          ? HYPHEN_WITH_SPACE + applicationDetail.applicationNumber
```

Comment:
Will there be an application here without number? We can skip the expression if there is no such possibility.

Reply:
https://github.com/bcgov/SIMS/pull/1914#discussion_r1184417241